sass/patterns/_search-page.scss

```diff
@@ -78,12 +78,12 @@
             text-shadow: none;
         }
         input {
-            background-color: $link-colour;
+            background-color: #034da1;
         }
         &--sort-form {
             text-align: right;
             input {
-                background-color: $link-colour;
+                background-color: #034da1;
             }

```

Comment:
It might be better to change these colours to use the $cta-blue variable, rather than hard code them.